sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -79,7 +79,6 @@ public partial class ClientRetryPolicy : System.ClientModel.Primitives.PipelineP
     {
         public static readonly System.ClientModel.Primitives.ClientRetryPolicy Default;
         public ClientRetryPolicy(int maxRetries = 3) { }
-        public System.TimeSpan GetNextDelay(System.ClientModel.Primitives.PipelineMessage message, int tryCount) { throw null; }
         protected virtual System.TimeSpan GetNextDelayCore(System.ClientModel.Primitives.PipelineMessage message, int tryCount) { throw null; }
```

Comment:
Can we remove the Core suffix now that we have one method?

Reply:
I think we can, actually ... looking